project/plugins.sbt

```diff
@@ -1,11 +1,12 @@
-addSbtPlugin("com.github.sbt" % "sbt-release" % "1.1.0")
-addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "1.0.1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")
-addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.2")
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0")
+addSbtPlugin("com.47deg" % "sbt-microsites" % "1.3.4")
```

Comment:
Why adding microsites plugins?

Reply:
cargo culting.  Looks like it was `addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")` that I needed to add back instead.